spec/consensus/light/accountability.md

```diff
@@ -0,0 +1,319 @@
+# Fork accountability 
+
+## Problem Statement
+
+Tendermint consensus guarantees the following specifications for all heights:
+* agreement -- no two correct full nodes decide differently.
+* validity -- the decided block satisfies the predefined predicate *valid()*.
```

Comment:
Nit regarding formatting: should we format this as `valid()` instead of *valid()*? (consistently, here and everywhere else). 

```suggestion
* validity -- the decided block satisfies the predefined predicate *valid()*.
```